### PR TITLE
feat: Cooperations

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**
Models Cooperations between Lernfair and some company, whose employees sign up as helpers.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -274,46 +284,46 @@ model log {
 }
 
 model match {
-  id                       Int               @id(map: "PK_92b6c3a6631dd5b24a67c69f69d") @default(autoincrement())
+  id                    Int       @id(map: "PK_92b6c3a6631dd5b24a67c69f69d") @default(autoincrement())
```
Comment (Jonasdoubleyou) [inline prisma/schema.prisma:287]: (sorry for the large diff, reformated with `prisma format`)

File: prisma/migrations/20230911183907_cooperations/migration.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+-- AlterTable
+ALTER TABLE "student" ADD COLUMN     "cooperationID" INTEGER;
+
+-- CreateTable
+CREATE TABLE "cooperation" (
+    "id" SERIAL NOT NULL,
+    "tag" TEXT NOT NULL,
```
Comment (dhenkel92) [inline prisma/migrations/20230911183907_cooperations/migration.sql:7]: Do we also need a Name that can be shown in the UI later on?

**Review by dhenkel92 — DISMISSED** [review]
👍

File: prisma/migrations/20230911183907_cooperations/migration.sql (hunk 2 above)
Comment (Jonasdoubleyou) [reply prisma/migrations/20230911183907_cooperations/migration.sql:7]: True!

**Review by dhenkel92 — APPROVED** [review]

**Pull request merged**